litentry/litentry-parachain
# initial implementation of drop3 pallet

resolves #273 

This PR:
- implements `pallet-drop3`, see comments in `src/lib.rs` for notes
- add matching unittest and benchmarking
- add locally generated weights.rs as pallet-benchmarked weights

Todos:
- merge to `drop3_testnet` branch once approved
- deploy the drop3 testing network
- see TODOs in codes
